pkg/contracts/contracts.go

```diff
@@ -0,0 +1,21 @@
+package contracts
+
+import (
+	"context"
+
+	"github.com/clearchanneloutdoor/io-sdk-golang/pkg/clients"
+	"golang.org/x/oauth2/clientcredentials"
+)
+
+const (
+	serverUrl string = "https://direct.cco.io"
```

Comment:
do we need to do any word to direct.cco.io to support contract-api? Or do we get that integration for free?

Reply:
not real sure. @cco-joshua ?

Reply:
Nothing actually exposed via direct.cco.io unless it's configured within the endpoints configuration. direct.cco.io was designed, essentially, for external partners, like agencies, to get access to data that they can work with or that is pertinent to them. We probably won't expose contracts to agencies unless there is a need to do so, so I'm not sure if we'll put it in direct.cco.io or not... or we still can, and simply choose not to give the clientID we configured for each agency access to the endpoint (we control which APIs and endpoints that the clients can access via clients configuration for direct.cco.io).

Ultimately, though, for our internal use cases, this is 100% fine. The client accepts a variadic string variable to override the host at the time we instantiate it. We'll leverage that to specify the server as appropriate at runtime so that the default host here is not used. Either way, the SDK will handle the 404 gracefully.